packages/client/src/objectSet/ObjectSetListenerWebsocket.ts

```diff
@@ -273,12 +286,19 @@ export class ObjectSetListenerWebsocket {
     const subscribe: ObjectSetStreamSubscribeRequests = {
       id,
       requests: readySubs.map<ObjectSetStreamSubscribeRequest>((
-        { objectSet, requestedProperties, requestedReferenceProperties },
+        {
+          objectSet,
+          requestedProperties,
+          requestedReferenceProperties,
+          interfaceApiName,
+        },
       ) => {
         return {
           objectSet: objectSet,
-          propertySet: requestedProperties,
-          referenceSet: requestedReferenceProperties,
+          propertySet: interfaceApiName == null ? requestedProperties : [],
```

Comment:
Empty set because OSW does not currently load SPT mappings. So currently with interfaces, we can say we don't support specifying properties and will load all properties. Reference properties TBD